src/views/Tree/ImportTree.js

```diff
@@ -0,0 +1,149 @@
+/** Build an import tree from a relative path or remote HTTP URL.
```

Comment:
If you could please, move this folder to the `components` folder, because this is not an actual route.